core/modules/order/components/UserOrdersHistory.ts

```diff
@@ -26,6 +27,13 @@ export default {
   methods: {
     onBottomScroll () {
       ++this.pagination.current
-    }
+      let total_count = this.$store.state.user.orders_history.total_count ? this.$store.state.user.orders_history.total_count : 0
+      if (Math.ceil(total_count / this.pagination.perPage) >= this.pagination.current) {
```

Comment:
What about changing it to:
```js
const isLastPage = this.pagination.current > Math.ceil(total_count / this.pagination.perPage);
if (!isLastPage) {
```

Then everyone would understand what this condition means even with 0 information about Math.ceil and these variables

Reply:
done